content/docs/reference/pkg/kubernetes/admissionregistration/v1/mutatingwebhookconfigurationlist.md

```diff
@@ -13,6 +13,7 @@ meta_desc: "Explore the MutatingWebhookConfigurationList resource of the admissi
 MutatingWebhookConfigurationList is a list of MutatingWebhookConfiguration.
 
 
+
```

Comment:
Any reason for this extra new line here and the others in other markdown files?

Reply:
Not sure where that came from. Something with the common docs code, I think.